cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/PhaseTapChangerConversion.java

```diff
@@ -130,11 +130,15 @@ private void addStepsFromTable(PhaseTapChangerAdder ptca) {
         String tableId = p.getId("PhaseTapChangerTable");
```

Comment:
Maybe you should also test if `tableId` is `null`, it is optional for `PhaseTapChangerTabular` according to the spec.